odl/tomo/geometry/conebeam.py

```diff
@@ -308,11 +326,11 @@ def det_refpoint(self, angle):
         """
         angle = float(angle)
         if angle not in self.motion_params:
-            raise ValueError('angle {} not in the valid range {}.'
+            raise ValueError('angle {} is not in the valid range {}.'
                              ''.format(angle, self.motion_params))
 
-        # ASTRA 'cone_vec' convention
-        return self.det_radius * np.array([-np.sin(angle), np.cos(angle), 0])
+        offset = self.det_radius * self._source_to_detector
```

Comment:
Name "offset" is slightly confusing. This is supposed to be the not-yet-rotated reference point, there must be some word for this, maybe `initial_refpoint`.